bpmn/hzvprocess.bpmn

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
💭 - The process looks awesome! I am just thinking about whether we should already set the technical properties in here. (e.g. defining type, retries or messages)